### DISCUSSION TITLE
How to set permissions to null via imported permissions file

### DISCUSSION THREAD

**Discussion by itsyosef** [directus/directus#17420] [Archive]
I'm trying to post to the "/utils/import/directus_permissions" endpoint to set up a permissions migration system between our development environments, and I'm having trouble figuring out how to handle null permissions.

On the UI, this is the 🚫 symbol. An example scenario I'm having trouble with is this: say I have a 🚫 on dev and I want to sync dev with staging, but the same role/action/collection combo has a more permissive permission rule on staging. Pulling the dev permissions from the "/permissions" endpoint will not result in an entry for that item since it is null there. This means either this permission will be missed in my migration, or I have to create a new entry to be uploaded that sets that permission as 🚫. I can't seem to figure out what that value should be/if this is possible. 

Does anyone have insight on this?

Thanks!

**Comment by rijkvanzanten** [comment]
The 🚫 permissions are effectively "There's no permissions row". To replicate that through the API, you have to delete any permissions row that was there before.

**Reply by itsyosef** [reply]
Thanks!
